### PR TITLE
Feature/feedback : #167 done 

### PR DESCRIPTION
**Pull request by VinayLodhi1712**
Feedback form backend and frontend both done successfully.
#167 done 

Backend successfully implementation

Frontend successfully implementation
https://github.com/user-attachments/assets/ffdc72ff-8ab5-496a-add8-0a1b566eaa19

Changes you need to do for successfully getting response on email -
smtp

Main things you need to care to making it correctly -

1. add your backend URL here in contact.jsx file in fetch request in place of "http://localhost:8080/" (if does not work in case with localhost)

2. insert your email and passkey/ if you have 2-factor authentication is on and generate a passkey and put it in the .env file here.

plz give level 3 as everyone give in contact us form its just request.

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -19,20 +19,20 @@ app.use(express.urlencoded({ limit: "10mb" }));
 const allowedOrigins = [
     "https://delightful-daifuku-a9f6ea.netlify.app",
     /https:\/\/deploy-preview-\d+--delightful-daifuku-a9f6ea\.netlify\.app/,
+    "http://localhost:5173"
 ];
```
Comment (AlfiyaSiddique) [inline backend/server.js:23]: Remove This localhost from allowedOrigins

File: frontend/src/App.css [hunk 2]
```diff
@@ -55,3 +55,57 @@
...
+
+@media (max-width: 600px) {
+  ::-webkit-scrollbar {
+    width: 7px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    background: #B91C1C;
+  }
+}
```
Comment (AlfiyaSiddique) [inline frontend/src/App.css:111]: @VinayLodhi1712 why the styles here for the scroll bar is repeated ex: ::-webkit-scrollbar

**Comment by AlfiyaSiddique** [comment]
@VinayLodhi1712 Some comments please fix it.
Also please let me know how to generate or get the smtp_password. Any doc or reference will be helpful

File: frontend/src/App.css (hunk 2 above)
Comment (VinayLodhi1712) [reply frontend/src/App.css:111]: i can correct it i think forget pull the changes before pushing so it merge the all changes.

**Comment by VinayLodhi1712** [comment]
i have change everything just you can check once now @AlfiyaSiddique 
also i am just asking like while working on frontend we work on url like localhost:3000 for fetch backend but here already backendURL defined which is i think deployed URL so i have to push with the localhost or backendURL.

backend

**Comment by AlfiyaSiddique** [comment]
@VinayLodhi1712 Do not worry about the backend URL value just use the variable for it everywhere and it will be changed in deployment according to its settings.
Also please update the PR

**Comment by AlfiyaSiddique** [comment]
Also, let me know about how to generate or get the smtp_password or Any doc or reference that will be helpful

**Comment by VinayLodhi1712** [comment]
well i have written that in .env.sample file and morely you can also do this
#To create a passkey on the phone or computer you’re on:
1. Go to https://myaccount.google.com/signinoptions/passkeys.
2. Tap Create a passkey and then Continue.(You'll be required to unlock your device.) 
3. A 16 character passkey is generated which you can use.

Also, To create passkeys on multiple devices, repeat these steps from those devices.

**Pull request merged**